account_invoice_customer_no_autofollow/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Invoice Customer No Autofollow",
+    "summary": "Do not add customer as follower in Invoices",
+    "author": "Cetmix,Odoo Community Association (OCA)",
+    "version": "16.0.1.0.0",
+    "website": "https://github.com/OCA/account-invoicing",
+    "category": "Account",
+    "depends": ["account"],
+    "maintainers": ["dessanhemrayev", "CetmixGitDrone"],
```

Comment:
Pls remove these maintainers or replace with the ones you would like to add.

Reply:
Done